packages/aws-cdk-lib/aws-codestarnotifications/README.md

```diff
@@ -12,6 +12,20 @@ where you want to receive them.
 Notification targets are objects that implement the `INotificationRuleTarget`
 interface and notification source is object that implement the `INotificationRuleSource` interface.
 
+### notificationRuleName
+You can set a name for the notification rule. Notification rule names must be unique in your AWS account.
+
+### enabled
+You can set the status of the notification rule.If the enabled is set to DISABLED, notifications aren't sent for the notification rule.The default is true.
+
+### detailType
+You can set the level of detail to include in the notifications for this resource.
+BASIC will include only the contents of the event as it would appear in AWS CloudWatch.
+FULL will include any supplemental information provided by AWS CodeStar Notifications and/or the service for the resource for which the notification is created.
+
+### createdBy
+You can set a name or email alias of the person who created the notification rule.If not specified, it means that the creator's alias is not provided.
+
```

Comment:
Could you please add sample code for the construct?

Also, I think that these `notificationRuleName`, `enabled`, `detailType` and `createdBy` blocks are the same as JSDoc for the properties in the construct, so they could be erased. The sample code could cover them.

``````suggestion
```ts
// sample conde
...
```
``````

Reply:
Sorry, it seems that there is already be the example, so it might be good to add the properties here.

https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/aws-codestarnotifications/README.md#examples

Reply:
@go-to-k 
Thank you for the review! 
I've made the changes!
By the way, even though I haven't specifically changed the integ-test, the CI process "request-cli-integ-test" is failing. Do you know why that might be?😅

Reply:
@yuppe-kyupeen 

Thanks for the change!

> By the way, even though I haven't specifically changed the integ-test, the CI process "request-cli-integ-test" is failing. Do you know why that might be?😅

Pulling from the main branch or pushing a new commit and running CI again often fixes it!

Reply:
@go-to-k 
It's resolved! Thank you🙏